catalog-info.yaml

```diff
@@ -1,9 +1,13 @@
+# ***********************************
+# SECTION START: Backstage logstash-filter-elastic-integration component
+# ***********************************
+
 ---
-# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json
+# yaml-language-server: $schema=https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/e57ee3bed7a6f73077a3f55a38e76e40ec87a7cf/rre.schema.json
```

Comment:
We should be using the latest version of the RRE schema, as it was before: https://gist.githubusercontent.com/elasticmachine/988b80dae436cafea07d9a4a460a011d/raw/rre.schema.json so that it reflects the latest changes.
For more details see (the internal link) https://elastic.slack.com/archives/C0522G6FBNE/p1700477179859799 and https://elasticco.atlassian.net/browse/CI-322?focusedCommentId=96184